deploy_nixos/nixos-instantiate.sh

```diff
@@ -8,17 +8,31 @@ config_pwd=$3
 shift 3
 
 # Building the command
+nixExpression=<<EOF
+
+EOF
+
 command=(nix-instantiate --show-trace --expr '
-  { system, configuration, ... }:
+  { system, configuration, hermetic ? false, ... }:
   let
-    os = import <nixpkgs/nixos> { inherit system configuration; };
-    inherit (import <nixpkgs/lib>) concatStringsSep;
+    os =
+      if hermetic
+        then import configuration
+        else import <nixpkgs/nixos> { inherit system configuration; };
+
+    inherit (os.pkgs) lib;
```

Comment:
Using `builtins.concatStringsSep` might be enough since it's the only lib function that is used below.

Reply:
TIL that concatStringsSep is a built-in! That's definitely preferred IMHO.